test/WebSites/FiltersWebSite/AuthorizeBasicMiddleware.cs

```diff
@@ -15,10 +16,8 @@ public class BasicOptions : AuthenticationOptions
     {
```

Comment:
This file contains 3 classes, which is something we do not follow. (even thought this is not directly related to your change ) could you please split this file into 3?